docs/releases/1.11.txt

```diff
@@ -720,6 +720,8 @@ Miscellaneous
 
 * Support for ``cx_Oracle`` < 5.2 is removed.
 
+* Added support for ``cx_Oracle`` 5.3.
```

Comment:
This could go under "Database backends" minor features. This section is backwards-incompatible changes.